test/acceptance/features/steps/steps.py

```diff
@@ -436,3 +437,34 @@ def update_binding_secret(context):
     output = openshift.apply(yaml.dump(secret_yaml), context.namespace.name)
     result = re.search(rf'.*{context.sb_secret}.*(created|unchanged|configured)', output)
     assert result is not None, f"Unable to apply YAML for binding secret '{context.sb_secret}': {output}"
+
+
+@then(u'cluster role "{cluster_role_name}" is available in the cluster')
+def check_cluster_role_exists(context, cluster_role_name):
+    openshift = Openshift()
+    roles_present, exit_code = openshift.is_resource_in("clusterroles")
```

Comment:
instead of getting all cluster roles, and then checking if `cluster_role_name` is present, we can try to get read directly `cluster_role_name`. If not found, we are going to get exit code different from 0.

you can use here `openshift.get_json_resource` instead.